README.md

```diff
@@ -147,7 +147,7 @@ If no user configuration is found, a set of default values will be used.
   "gitmoji": {
     "autoAdd": false,
     "emojiFormat": "code | emoji",
-    "scopePrompt": false,
+    "scopePrompt": false | ["feat", "fix", "ci", "build"],
```

Comment:
This always references to the default values so probably it's worth it keeping it as a boolean and mention this somewhere else